browser-test/src/applicant/applicant_question_translation.test.ts

```diff
@@ -0,0 +1,182 @@
+import {test, expect} from '../support/civiform_fixtures'
+import {
+  loginAsAdmin,
+  logout,
+  selectApplicantLanguage,
+  validateScreenshot,
+  validateToastMessage,
+} from '../support'
+
+test.describe(
+  'Admin can manage translations',
+  {tag: ['@uses-fixtures']},
+  () => {
+    test('creates a question and adds translations', async ({
```

Comment:
suggestion (blocking): Edit the test names so they match what we want to check here. Eg. this one could be something like "Expect translations to show up for applicants".